api/internal/localizer/localizer.go

```diff
@@ -368,6 +425,91 @@ func (lc *localizer) localizeDir(path string) (string, error) {
 	return locPath, nil
 }
 
+// copyChartHome copies helm chart home path, relative to root that lc is at, to
+// the same location relative to dst and returns said relative copied location.
+//
+// If path indicates the default chart home, copyChartHome copies chart home to
+// the same location in dst, without following symlinks.
+func (lc *localizer) copyChartHome(path string) (string, error) {
+	if filepath.IsAbs(path) {
+		return "", errors.Errorf("absolute path %q not handled in alpha", path)
+	}
+	locPath, err := filepath.Rel(lc.root.String(), lc.root.Join(path))
+	if err != nil {
+		return "", errors.WrapPrefixf(err, "no path from root to chart home")
+	}
+	if path == "" {
+		path = types.HelmDefaultHome
+		locPath = ""
+	}
+	// chart home may serve as untar destination
+	if !lc.fSys.Exists(lc.root.Join(path)) {
+		return locPath, nil
+	}
+	// perform localize directory checks
+	ldr, err := lc.ldr.New(path)
+	if err != nil {
+		return "", errors.Wrap(err)
+	}
+	clean, err := filesys.ConfirmDir(lc.fSys, ldr.Root())
+	if err != nil {
+		log.Panicf("unable to establish validated directory %q: %s", ldr.Root(), err)
+	}
+	var copyDst string
+	isDefault := lc.root.Join(path) == lc.root.Join(types.HelmDefaultHome)
```

Comment:
Seems like it would be sufficient to just check if `path == types.HelmDefaultHome`. The extra calls to `lc.root.Join` doesn't seem to be adding anything to this equality check

Reply:
`lc.root.Join` will account for `path`s that lead to `charts`, such as `./charts`.